qiskit_nature/circuit/library/ansatzes/ucc.py

```diff
@@ -414,11 +419,17 @@ def _build_fermionic_excitation_ops(self, excitations: Sequence) -> List[Fermion
                 label[occ] = "+"
             for unocc in exc[1]:
                 label[unocc] = "-"
+            # Real part of the wave-function 
             op = FermionicOp("".join(label), display_format="dense")
-            op -= op.adjoint()
-            # we need to account for an additional imaginary phase in the exponent (see also
+            op_adj = op.adjoint()
+            op_minus = 1j*(op - op_adj)
+            # we need to account for an additional imaginary phase in the exponent hence multiplied by 1j (see also
             # `PauliTrotterEvolution.convert`)
-            op *= 1j  # type: ignore
-            operators.append(op)
+            operators.append(op_minus)
+            
+            # Appending the corresponding imaginary part
+            if self._include_imaginary:
+                op_plus = op + op_adj
```

Comment:
Suggestion: `op_plus = -1*(op+op_adj)`

While the -1 will get incorporated in parameter optimization, the negative sign is consistent with `PauliTrotterEvolution` class

Reply:
fixed in 21a3456